rust/actyx/ax/tests/version_compat.rs

```diff
@@ -165,14 +164,14 @@ fn setup() -> &'static Binaries {
 }
 
 fn download(package: &str, bin: &str, version: Version, dst_dir: &Path, may_skip: &mut bool) -> Option<PathBuf> {
-    let arch = match Arch::current() {
-        Arch::x86_64 => "amd64",
-        Arch::aarch64 => "arm64",
-        Arch::arm => "arm",
-        Arch::armv7 => "armhf",
-        x => panic!("unsupported arch: {}", x),
-    };
-    let name = format!("{}-{}-linux-{}", package, version, arch);
+    // This is a bit of an abuse but we're currently running this only on 64-bit Linux
+    // hence taking the liberty to "force" this here. For a reference, the previous
+    // mapping was:
```

Comment:
I’d prefer keeping the mapping in here, match on the `ARCH` using string literals

Reply:
The mapping is misleading in our process because it doesn't do anything as I explained in the comment. I'll bring it back though